examples/templates/docker/README.md

```diff
@@ -6,19 +6,22 @@ tags: [local, docker]
 
 # docker
 
-## Getting started
-
-Run `coder templates init` and select this template. Follow the instructions that appear. 
+To get started, run `coder templates init`. When prompted, select this template.
+Follow the on-screen instructions to proceed.
 
 ## Adding/removing images
 
-After building and pushing an image to an image registry (e.g., DockerHub), you can edit the template to make the image available to users. 
-
-Edit the template:
+After building and pushing an image to an image registry (e.g., DockerHub), edit
+the template to make the image available to users:
 
 ```sh
+# Open the template
 vim main.tf
 ```
+
+Modify your file to match the following:
+
+```dockerfile
```

Comment:
I'm using GitHub's "diff" view to indicate the changes made. If you'd prefer not to use it, this is HCL

```suggestion
```diff
```

or 

```suggestion
```hcl
```